### PR TITLE
Add Index of words and commit other Follow/Unfollow butons updated files

### PR DESCRIPTION
**Pull request by gravidesaldrin**
Hi Sir. Please review this PR. Thank You po.

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -6,6 +6,17 @@ class Category < ActiveRecord::Base
 
   validates :name, presence: true, uniqueness: true
 
+  def search_words current_filter, current_user
```
Comment (MuhammadTamzid) [inline app/models/category.rb]: why word search method in category model?

File: app/models/word.rb [hunk 2]
```diff
@@ -10,4 +10,9 @@ class Word < ActiveRecord::Base
 
   accepts_nested_attributes_for :word_answers, allow_destroy: true,
     reject_if: lambda {|attribute| attribute[:content].blank?}
+
+  def correct_content
+    self.word_answers.select{|attribute| attribute.correct}
+    .map{|attribute| attribute.content}.join(", ")
```
Comment (MuhammadTamzid) [inline app/models/word.rb]: indent

File: app/views/admin/users/edit.html.erb [hunk 3]
```diff
@@ -1,4 +1,9 @@
 <% provide :title, t(".title") %>
+<ul class="breadcrumb">
+    <li><%= link_to t(".current_menu"), admin_users_path %></li>
```
Comment (MuhammadTamzid) [inline app/views/admin/users/edit.html.erb]: indent
two space

File: app/views/admin/users/edit.html.erb [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
+    <li><%= link_to @user.name, admin_user_path @user %></li>
```
Comment (MuhammadTamzid) [inline app/views/admin/users/edit.html.erb]: same above

File: app/views/admin/users/edit.html.erb [hunk 5]
```diff
@@ -1,4 +1,9 @@
...
+    <li class="active"><%= t ".current_action" %></li>
```
Comment (MuhammadTamzid) [inline app/views/admin/users/edit.html.erb]: same above

File: app/views/admin/users/index.html.erb [hunk 6]
```diff
@@ -1,4 +1,8 @@
 <% provide :title, t(".title") %>
+<ul class="breadcrumb">
+    <li><%= link_to t(".title"), admin_users_path %></li>
```
Comment (MuhammadTamzid) [inline app/views/admin/users/index.html.erb]: indent

File: app/views/admin/users/index.html.erb [hunk 7]
```diff
@@ -1,4 +1,8 @@
...
+    <li class="active"><%= t ".current_action" %></li>
```
Comment (MuhammadTamzid) [inline app/views/admin/users/index.html.erb]: indent

File: app/views/admin/users/show.html.erb [hunk 8]
```diff
@@ -1,4 +1,8 @@
 <% provide :title, @user.name %>
+<ul class="breadcrumb">
+    <li><%= link_to t(".current_menu"), admin_users_path %></li>
```
Comment (MuhammadTamzid) [inline app/views/admin/users/show.html.erb]: indent

File: app/views/admin/users/show.html.erb [hunk 9]
```diff
@@ -1,4 +1,8 @@
...
+    <li class="active"><%= t ".current_action" %></li>
```
Comment (MuhammadTamzid) [inline app/views/admin/users/show.html.erb]: indent

File: app/controllers/words_controller.rb [hunk 10]
```diff
@@ -0,0 +1,11 @@
+class WordsController < ApplicationController
+  def index
+    @category = Category.first
```
Comment (MuhammadTamzid) [inline app/controllers/words_controller.rb]: ??

File: app/models/word.rb [hunk 11]
```diff
@@ -10,4 +10,20 @@ class Word < ActiveRecord::Base
 
   accepts_nested_attributes_for :word_answers, allow_destroy: true,
     reject_if: lambda {|attribute| attribute[:content].blank?}
+
+  def correct_content
+    self.word_answers.select{|attribute| attribute.correct}
+      .map{|attribute| attribute.content}.join(", ")
+  end
+
+  def Word.search_words category, current_filter, current_user
```
Comment (dieunb) [inline app/models/word.rb]: check convention to write class method

File: app/controllers/words_controller.rb [hunk 12]
```diff
@@ -0,0 +1,12 @@
+class WordsController < ApplicationController
+  def index
+    @categories = Category.all
+    if params[:category]
+      @category = Category.find_by id: params[:category]
```
Comment (dieunb) [inline app/controllers/words_controller.rb]: `@categories.find_by id: params[:category_id]`

File: app/controllers/words_controller.rb [hunk 13]
```diff
@@ -0,0 +1,12 @@
+class WordsController < ApplicationController
+  def index
+    @categories = Category.all
+    if params[:category]
```
Comment (dieunb) [inline app/controllers/words_controller.rb]: why not `params[:category_id]`

**Pull request merged**